FileShareClient/Models/Attributes.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace UKHO.FileShareClient.Models
+{
+    public class Attributes : IEquatable<Attributes> 
```

Comment:
Please rename `Attributes` class to `BatchAttributesSearchAttribute` (singular).

Reply:
Updated to SearchBatchAttribute.